docs/examples/dvs-from-file.ipynb

```diff
@@ -0,0 +1,318 @@
+    "dvs_events.write_file(events_file_name)\n",
+    "print(\"Wrote %r\" % events_file_name)\n",
+    "del dvs_events"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can view the data by using the `DVSEvents` class to load the events, group the events\n",
+    "into frames, and then make the frames into a video with the help of Matplotlib's\n",
+    "animation support."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dvs_events = nengo_loihi.dvs.DVSEvents.from_file(events_file_name)\n",
+    "\n",
+    "dt_frame_us = 20e3\n",
+    "t_frames = dt_frame_us * np.arange(int(round(t_length_us / dt_frame_us)))\n",
+    "\n",
+    "fig = plt.figure()\n",
+    "imgs = []\n",
+    "for t_frame in t_frames:\n",
+    "    t0_us = t_frame\n",
+    "    t1_us = t_frame + dt_frame_us\n",
+    "    t = dvs_events.events[:][\"t\"]\n",
+    "    m = (t >= t0_us) & (t < t1_us)\n",
+    "    events_m = dvs_events.events[m]\n",
+    "\n",
+    "    # show \"off\" (0) events as -1 and \"on\" (1) events as +1\n",
+    "    events_sign = 2.0 * events_m[\"p\"] - 1\n",
+    "\n",
+    "    frame_img = np.zeros((dvs_height, dvs_width))\n",
+    "    frame_img[events_m[\"y\"], events_m[\"x\"]] = events_sign\n",
+    "\n",
+    "    img = plt.imshow(frame_img, vmin=-1, vmax=1, animated=True)\n",
+    "    imgs.append([img])\n",
+    "\n",
+    "del dvs_events\n",
+    "\n",
+    "ani = ArtistAnimation(fig, imgs, interval=50, blit=True)\n",
+    "HTML(ani.to_html5_video())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Nengo network\n",
+    "\n",
+    "We can now load our data into a NengoLoihi network, using the\n",
+    "`nengo_loihi.DVSFileChipNode` type of `nengo.Node`. We pass the node the path to our\n",
+    "data file. We also use the optional `pool` parameter to pool over space. This allows our\n",
+    "node to output at a lower spatial resolution than the standard 180 x 240 pixel DVS\n",
+    "output. We then make two ensembles: one to receive the positive polarity \"on\" events,\n",
+    "and the other to receive the negative polarity \"off\" events. Since we passed\n",
+    "`channels_last=True` to our node, the polarity will be the least-significant index of\n",
+    "our node output, and we connect it to the ensembles accordingly (by sending all even\n",
+    "outputs to the negative polarity ensemble `ensembles[0]`, and the odd outputs to the\n",
+    "positive polarity ensemble `ensembles[1]`). We run the simulation and collect the spikes\n",
+    "from both the negative and positive polarity ensembles."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pool = (10, 10)\n",
+    "\n",
+    "height = 180 // pool[0]\n",
+    "width = 240 // pool[1]\n",
+    "gain = 101\n",
```

Comment:
I just had it at 100 at first, and then found that 101 gives slightly better results. I think it's because 100 can often result in values that are right around the firing threshold, where as 101 more consistently pushes things over the firing threshold (e.g. if a neuron has a firing threshold of 200, an input of 100 might put it at but not over the firing threshold after two timesteps, causing it to fire every three timesteps, whereas with the higher gain, the input would be 101 and thus consistently put it over).